hooks/post_gen.dart

```diff
@@ -55,16 +55,18 @@ Future<void> _runAmplifyInit(HookContext context) async {
     'amplify',
     ['init', '--yes'],
     workingDirectory: '{{project_name}}',
+    mode: ProcessStartMode.inheritStdio,
   );
 
+  // Complete progress so it doesn't interfere with stdout of `amplify`
+  amplifyInitProgress.complete();
+
   final exitCode = await result.exitCode;
 
   if (exitCode == 0) {
-    amplifyInitProgress.complete('Project is initialized.');
+    context.logger.success('Project is initialized.');
   } else {
-    final errorBytes = await result.stderr.first;
-    final error = systemEncoding.decode(errorBytes);
-    amplifyInitProgress.complete('Project initialization failed. $error');
+    context.logger.err('Project initialization failed.');
```

Comment:
Out of curiosity: Will this share the error message as well?

Reply:
Yep! The child process will inherit both stdout and stderr. Here's how a failure looks

<img width="477" alt="Screen Shot 2022-08-04 at 4 25 34 PM" src="https://user-images.githubusercontent.com/24740863/182971739-92ee8059-680f-476c-a34e-aa545c796a48.png">

Reply:
This is awesome! Thank you!